pkg/NAMESPACE

```diff
@@ -127,10 +127,17 @@ exportClasses("Column")
 
 exportMethods("asc",
               "desc",
-              "first",
               "last",
               "lower",
               "upper",
+              "like",
```

Comment:
Ok - It'll be great if you could also sort the generics file (sorted per section)